src/App.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useCallback, useEffect } from 'react'
 import 'antd/dist/antd.min.css'
 import './App.scss'
+import './pages/components/header/sidebar/sidebar.scss'
```

Comment:
please move that import to the `SideBar.tsx` file

Reply:
@NoamGaash 
Sorry. My mistake. The mentioned import of `sidebar.scss` is basically redundant inside `App.tsx` because it is already imported as part of `SideBar.tsx` .

Also, I removed unused import of `React from 'react'`.

![image](https://github.com/hasadna/open-bus-map-search/assets/52656401/451b81ba-3315-4fae-9c1c-254685aebce0)

Reply:
@AlexKomanov thanks!
regarding React import, it is important:
![image](https://github.com/hasadna/open-bus-map-search/assets/11145132/5fcca845-c92e-4c06-816e-50e10869e9f4)
you should import react to use tags in your tsx files

Reply:
Oh. Got it.